tests/decodehelp.h

```diff
@@ -49,7 +49,7 @@ uint32_t dumpFourcc = VA_FOURCC_I420;
 char *inputFileName = NULL;
 int renderMode = 1;
 static int32_t waitBeforeQuit = 1;
-
+static uint32_t frameCount = 0, useSpecified = 0;
```

Comment:
one variable is enough, you can name it frameCountMax and init it to numeric_limits<uint32_t>::max()